test/err.js

```diff
@@ -1,27 +1,86 @@
 var expect = require("chai").expect;
```

Comment:
Can we please use single quotes throughout? We should probably use a linter for this eventually. Consistency. :)

Something to clean up later.